app/views/pages/about.html.erb

```diff
@@ -2,68 +2,90 @@
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "What’s on Find open data",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">You can find:</p>
+
+      <%= render "govuk_publishing_components/components/list", {
+        visible_counters: true,
+        items: [
+          "data published by central government, local authorities and public bodies",
+          "links to download data files",
+          "help on creating an account to publish data",
+        ]
+      } %>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "How Find open data and policy work together",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">Better data can help inform better policy-making and continuous improvement of services. Departments can combat
+        fraud and reduce waste.</p>
+
+      <p class="govuk-body">By taking ethical, open, innovative and transparent approaches to data we can build greater levels of trust with
+        citizens, whilst delivering more cost-effective and better targeted and tailored services to their needs.</p>
+
+      <p class="govuk-body">Data is at the heart of digital transformation and a part of the <%= link_to "Government Transformation Strategy", "https://www.gov.uk/government/publications/government-transformation-strategy-2017-to-2020/government-transformation-strategy-better-use-of-data", class: "govuk-link" %>.</p>
+
+      <p class="govuk-body">GDS works with other organisations both within and outside government on shaping the strategic direction on how
+        data is managed, accessed and used. We work collaboratively on a range of issues with the Office for National
+        Statistics, Department for Digital, Culture, Media & Sport and The National Cyber Security Centre.</p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Reuse information published on Find open data",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">You can republish any of the content on Find open data, as long as you meet the conditions of either the <%= link_to "Open Government Licence", "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/", class: "govuk-link" %> or
+        the licence it's covered by. For questions about a dataset, you can contact the publisher directly if they’ve
+        provided contact details on the relevant dataset page.</p>
+      
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Build services using registers",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">You can find <%= link_to "GOV.UK registers", "https://registers.cloudapps.digital/", class: "govuk-link" %> that are managed and approved by government and ready to use in your service.</p>
```

Comment:
The link at the moment 301s to the service domain, so we might as well remove a redirect for visitors.

Picky - I think that the `r` should be capitalised, since it's a name of a service and that's how it appears on the GOV.UK Registers site.

```suggestion
      <p class="govuk-body">You can find <%= link_to "GOV.UK Registers", "https://www.registers.service.gov.uk/", class: "govuk-link" %> that are managed and approved by government and ready to use in your service.</p>
```